src/library.js

```diff
@@ -2646,6 +2645,7 @@ LibraryManager.library = {
   },
 
   emscripten_run_script_string__sig: 'ii',
+  emscripten_run_script_string__deps: ['emscripten_builtin_malloc', 'emscripten_builtin_free'],
```

Comment:
I feel like we should document this usage of `emscripten_builtin_malloc` somehow. Perhaps a comment,
```cpp
// It is ok to leak this allocation, so use emscripten_builtin_malloc which skips LSan.
```

Another option would be to make an alias of it to `leaking_malloc`? Then the name would document the intent.

Reply:
Yes, I agree, we should probably change `emscripten_builtin_malloc` to something like `malloc_unchecked`.   I think that would be larger/wider change though.  For now I will add comments.